cloudconfig/cloudinit/interface.go

```diff
@@ -413,7 +411,7 @@ type HostnameConfig interface {
 // NetworkingConfig is the interface for managing configuration of network
 type NetworkingConfig interface {
 	// AddNetworkConfig adds network config from interfaces to the container.
-	AddNetworkConfig(interfaces corenetwork.InterfaceInfos) error
+	AddNetworkConfig(interfaces corenetwork.InterfaceInfos, matchHWAddr bool) error
```

Comment:
Is there some way we could do it as part of the Interface definition, instead of a separate bool?